client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -677,6 +678,17 @@ export class WorkspaceView extends React.Component<any, any> {
       ? this.getAvailablePointers(workspace, rootWorkspaceScratchPad)
       : this.getAvailablePointers(workspace);
 
+    const shouldShowJudgeTextFromScratchpad =
+      hasParent &&
+      !isOracleWorkspace &&
+      doesRootWorkspaceScratchpadContainRelevantContent;
+
+    const judgeRegex = /(?<=<judge>)(.*)(?=<judge>)/;
```

Comment:
Lookbehind `?<=` is not supported by Firefox, so Firefox crashes when loading a workspace.

I'm pretty sure you can change this regex to `/<judge>(.*)<judge>/`. And then the second item in the result array returned by `.exec` will be the match within the parentheses (i.e., the text in between the tags). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec

Reply:
Good call, will update. That regex would work, according to https://regexr.com/